tests/neptune_fastai/test_e2e.py

```diff
@@ -49,7 +54,8 @@ def is_cat(x):
 class TestE2E:
     def test_vision_classification(self):
```

Comment:
Should we update the test name to reflect that it is with handler

`test_vision_classification_with_handler`?